eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Scripts.hs

```diff
@@ -332,7 +333,7 @@ instance Era era => DecCBORGroup (AlonzoPlutusPurpose AsIx era) where
   decCBORGroup =
     decodeWord8 >>= \case
       0 -> AlonzoSpending . AsIx <$> decCBOR
-      1 -> AlonzoMinting . AsIx <$> decCBOR
+      1 -> trace "HERE!!!!" (AlonzoMinting . AsIx <$> decCBOR)
```

Comment:
It looks like this got left in by mistake :smile:

Reply:
haha, oops!  Thank you